src/database/connection.ts

```diff
@@ -0,0 +1,28 @@
+import { createConnection, getConnection } from 'typeorm';
+
+const connection = {
+  async create(callback: Function | null = null) {
+    try {
+      let connection = await createConnection();
+      if (callback) callback(connection);
+    } catch (e) {
+      console.log(e);
+    }
+  },
+
+  async close() {
+    await getConnection().close();
+  },
+
+  async clear() {
+    const connection = getConnection();
+    const entities = connection.entityMetadatas;
+
+    entities.forEach(async entity => {
+      const repository = connection.getRepository(entity.name);
+      await repository.query(`DELETE FROM public.${entity.tableName}`);
```

Comment:
I think you can use this approach to delete the entity
```
await connection.getRepository(Entity).createQueryBuilder().softDelete();
```

Reply:
Why soft delete? Wouldn't that make rows remain inside test DB forever?? 😱

Reply:
Yea, sorry I was confusing with the typeorm plus documentation(https://github.com/iWinston/typeorm-plus), I think you can delete in this way
```
await connection.createQueryBuilder()
    .delete()
    .from(Entity)
    .execute()
```

Reply:
I like this one 😃☝️

Reply:
Changed to `clear` that should be using `TRUNCATE`. 

```
TRUNCATE quickly removes all rows from a set of tables.
It has the same effect as an unqualified DELETE on each table,
but since it does not actually scan the tables it is faster.
```